spec/support/common_context.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.shared_context 'with local server', :with_local_miq_server do
+  before do
+    MiqRegion.seed
+
+    @zone = EvmSpecHelper.local_miq_server.zone
```

Comment:
Can we possibly use `rspec` constructs for this?

```ruby
RSpec.shared_context 'with local server', :with_local_miq_server do
  let(:local_miq_server) { EvmSpecHelper.local_miq_server }
  let(:zone)             { local_miq_server }

  before do
    MiqRegion.seed
    zone
  end
end
```

I just see that you are changing `zone` to `@zone` in some places, and this would be an option for avoiding that.

Reply:
2 places ignore the zone
8 places use `zone`
10 places use `@zone` (which I don't like)
others use both `miq_server` and `zone` but I didn't convert because this code only did `@zone`. (but those support the `let()` syntax)

Is the idea to avoid assigning the `@zone`? Or assign it and slowly go away from it? Not sure how the variable syntax would help us here

```ruby
    before do	
      _, _, @zone = EvmSpecHelper.create_guid_miq_server_zone	
    end
```